scripts/build.d/cmake

```diff
@@ -8,12 +8,12 @@ pkgname=cmake
 
 if [ "${SRCSYNC}" == "tarball" ] ; then
 
-  pkgver=${VERSION:-3.25.1}
+  pkgver=${VERSION:-3.27.9}
```

Comment:
Please use the latest 3.28.1

Reply:
Thanks for your comment. That specific version is based on the error message I saw when I tried to build modmesh.   There is a new command and it is included after 3.27. 3.27.9 is the latest version in 3.27. I haven't tried 3.28. Let me test if I can build modmesh with 3.28.